src/pyodide/internal/workers.py

```diff
@@ -920,7 +921,23 @@ def _wrap_subclass(cls):
         setattr(cls, k, _wrap_attr(v))
 
 
-class DurableObject:
+class EntrypointBase:
+    async def _wrapper_func(self, relaxed, inst, prop, *args, **kwargs):
```

Comment:
This should be a top level function, not a class method.

Reply:
I put it in the class so that I can easily access it from python-entrypoint-helper. It seems preferable over having to call `getPyodide` and importing the function manually, do you disagree?

Reply:
I think it's better to get it from top level so people can't break things by defining their own method with the same name.